.github/workflows/ci.yml

```diff
@@ -429,6 +429,36 @@ jobs:
         with:
           job-name: Account Console IT
 
+  forms-integration-tests:
+    name: Forms IT
+    runs-on: ubuntu-latest
+    needs: build
+    timeout-minutes: 75
+    strategy:
+      matrix:
+        browser: [chrome]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: integration-test-setup
+        name: Integration test setup
+        uses: ./.github/actions/integration-test-setup
+
+      - name: Run Account Console IT
+        run: ./mvnw test -Dsurefire.rerunFailingTestsCount=${{ env.SUREFIRE_RERUN_FAILING_COUNT }} -nsu -B -Pauth-server-quarkus -Dtest=**.forms.** -Dbrowser=${{ matrix.browser }} "-Dwebdriver.chrome.driver=$CHROMEWEBDRIVER/chromedriver" -f testsuite/integration-arquillian/tests/base/pom.xml | misc/log/trimmer.sh
```

Comment:
Does the Chrome driver need to be specified explicitly? There is no means of auto detecting it?

Reply:
Apparently yes, we're also specifying it in other testing stages like Account Console.